engine/src/main/java/org/terasology/rendering/nui/layers/ingame/PauseMenu.java

```diff
@@ -33,7 +31,6 @@ public void initialise() {
         WidgetUtil.trySubscribe(this, "settings", widget -> getManager().pushScreen("settingsMenuScreen"));
         WidgetUtil.trySubscribe(this, "mainMenu", widget -> CoreRegistry.get(GameEngine.class).changeState(new StateMainMenu()));
         WidgetUtil.trySubscribe(this, "exit", widget -> CoreRegistry.get(GameEngine.class).shutdown());
-        WidgetUtil.trySubscribe(this, "crashReporter", widget -> CrashReporter.report(new Throwable("Report an error."), LoggingContext.getLoggingPath()));
```

Comment:
I'd keep the button in the in-game menu - it's fairly well-hidden as is, no reason to obscure it.